### PR TITLE
Fixing3bugs

### PR DESCRIPTION
**Pull request by studentOfITC**
Create a review to allow me merge

### CODE REVIEW

File: FixedBugs.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixed Bug1, Bug2, Bug3 Bug4
```
Comment (Jesferred) [inline FixedBugs.txt:1]: good!

**Comment by Jesferred** [comment]
I allow

**Comment by MPTCourseTeacher** [comment]
> I allow

try to press "Add you review" button and select "Approve" option

**Review by MPTCourseTeacher — APPROVED** [review]
Thank you for making pull request. Please, wait for @Jesferred approve and you can merge

**Comment by MPTCourseTeacher** [comment]
@studentOfITC add you name in description section, please

**Review by Jesferred — APPROVED** [review]

**Comment by studentOfITC** [comment]
> @studentOfITC add you name in description section, please

Check [readme.md](https://github.com/studentOfITC/SPR3)

**Pull request merged**